src/Devices/PHProbe.h

```diff
@@ -1,13 +1,23 @@
 #pragma once
 #include "Arduino.h"
 
+/**
+ * Issuing the cal,mid command after the
+ * EZO pH circuit has been calibrated, will
+ * clear the other calibration points.
+ * Full calibration will have to be redone
```

Comment:
Is a "full calibration" the same as `twoPointCalibration()`? If so, we should say so. If not, how do we do a full calibration?